monitoring/scripts/comprehensive_test.py

```
@@ -0,0 +1,752 @@
+                logger.error(f"❌ Alert processing failed: {response.status_code}")
+            
+            # Test all endpoints
+            endpoints = ['/alerts', '/alerts/summary', '/alerts/history']
+            working_endpoints = 0
+            
+            for endpoint in endpoints:
+                try:
+                    response = requests.get(f'http://localhost:5001{endpoint}', timeout=10)
+                    if response.status_code == 200:
+                        working_endpoints += 1
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)